docs/examples.rst

```diff
@@ -0,0 +1,186 @@
+Examples
+########
+
+Flow Production Tracking Jira Bridge comes with several examples than can be found in
+the `examples <https://github.com/shotgunsoftware/sg-jira-bridge/tree/master/examples>`_ folder.
+
+These examples can be loaded if the `examples` directory is added to the PYTHONPATH in the
+`settings.py <https://github.com/shotgunsoftware/sg-jira-bridge/blob/master/settings.py#L95>`_ file.
+
+To properly use your custom syncer, you need to make sure you're correctly referencing it.
+This is done by using its name in the ``Jira Sync URL`` field of your Flow Production Tracking Project
+as well as in the Webhook URL on the Jira side.
```

Comment:
I think it's worth adding that the webhook should be updated to send timelog events as well

![image](https://github.com/shotgunsoftware/sg-jira-bridge/assets/123113322/bb05cf0d-d35c-4903-aadc-0b03912c77f1)